24_clustering.Rmd

```diff
@@ -18,7 +18,7 @@ features/samples that share pattern.  For example, with clustering, we
 can find group of samples that share similar community
 composition. There are multiple clustering algorithms available.
 
-## Introduction to bluster package
+## Introduction to bluster package and wrapper
 
 *bluster* is a Bioconductor package specialized in clustering. It offers 
```

Comment:
The *bluster* Bioconductor package provides specialized tools for clustering data in the `SummarizedExperiment` container.